src/net.h

```diff
@@ -168,18 +168,28 @@ enum class ConnectionType {
     ADDR_FETCH,
 };
 
+struct ConnCounts
+{
+    const int all = 0;
+    const int in = 0;
+    const int out = 0;
+    const bool tor_only = false;
+
+    ConnCounts()
```

Comment:
style: the compiler will implicitly define this default ctor. No need to do it yourself.